.github/workflows/main.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
 
 
       - name: Test and build
-        run: ./gradlew build
+        run: ./gradlew test installDist
```

Comment:
Hva er `installDist?`

Reply:
installDist bygger ikke en fat-jar slik som build, men bygger alle avhengighetene hver for seg i separate jars. Og da er det ikke lenger behov for noe shadow-jar-greier for å inkludere ting som default skippes i fat-jar-bygget, er slik jeg har forstått det

Reply:
Kan lite om jar, men høres bra ut!